Library/Formula/mono.rb

```diff
@@ -20,8 +20,8 @@ class Mono < Formula
   option "without-fsharp", "Build without support for the F# language."
 
   resource "monolite" do
-    url "http://storage.bos.xamarin.com/mono-dist-4.2.0-release/9b/9b990f2b19b1a534925cce3ddaabb70654b76066/monolite-135-latest.tar.gz"
-    sha256 "1529edbf34ebe498d315464e1211e65531ba25c492ba678a5bb079986a784131"
+    url "http://download.mono-project.com/monolite/monolite-138-latest.tar.gz"
+    sha256 "a7b1b0c20f28747e783d91f6e8e40e4c788bd25ca6192b43d622ea842d434a48"
   end
 
   resource "fsharp" do
```

Comment:
I don't know anything about the Mono/F# world; is there a reason we haven't moved to F# v4? https://github.com/fsharp/fsharp/releases